src/control/cmd/daos_agent/mgmt_rpc.go

```diff
@@ -52,7 +52,7 @@ type mgmtModule struct {
 
 func (mod *mgmtModule) HandleCall(session *drpc.Session, method int32, req []byte) ([]byte, error) {
 	switch method {
-	case drpc.MethodGetAttachInfo:
+	case int32(drpc.MethodGetAttachInfo):
```

Comment:
It seems like it would be better to do this conversion in `drpc/module_svc.go`. In the `ProcessMessage()` method it's already resolving the int32 method from the message to a `drpc.Method`, so it would be a small change to just adjust the `HandleCall()` interface to specify that it receives a `drpc.Method` instead of an int32. That way other module implementors don't need to deal with or know about the conversion.